### PR TITLE
Session9 clean code

### PR DESCRIPTION
**Pull request by andreigue**
Overview of Ch9 topics:

- Writing clean and understandable code.
- Commenting and documentation.
- Activity: Rewriting a given messy code snippet to make it clean and readable.

### CODE REVIEW

**Review by hpyhacking — CHANGES_REQUESTED** [review]
not complete

File: programming-fundamentals/chapter9.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
+---
+title: "Chapter 9: Best Practices and Readability"
+date: "2023-11-07"
+author: "Andrei Guevorkian"
+illustrator: "Dengyijia Liu"
+questions:
+  - function-use: |
+      When might you choose to create a function in your code instead of writing out the steps each time? Can you think of an example from a game or an app?
+  - commenting: |
+      3. What's the difference between a comment that is helpful and one that is unnecessary? Provide examples.
```
Comment (jp524) [inline programming-fundamentals/chapter9.md:10]: Remove "3." if not required.

File: programming-fundamentals/chapter9.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+
+When it comes to variables, the names should reflect their content or purpose as well. For instance:
+
+```typescript
+playerScore = 0
+```
+
+This variable clearly represents the player's score.
+
+> Note: In our made up pseudocode programming language, we have function names follow the `P`ascal`C`ase notation, and variables follow either camel`C`ase or snake`_`case, making sure to stay consistent throughout your code.
```
Comment (jp524) [inline programming-fundamentals/chapter9.md:70]: The code blocks make this sentence hard to read in my opinion. Could it be rewritten by showing an example of pascal case, camel case and snake case for the same variable name?

**Review by jp524 — CHANGES_REQUESTED** [review]

File: programming-fundamentals/chapter9.md (hunk 2 above)
Comment (andreigue) [reply programming-fundamentals/chapter9.md]: Edits made. Ready for review

**Review by jp524 — APPROVED** [review]

**Review by hpyhacking — APPROVED** [review]

**Pull request merged**